app/js/controllers/admin_user.js

```diff
@@ -62,9 +62,12 @@ function AdminUserCtrl($scope, account, $routeParams, $location) {
         }
         var query = $scope.nameQuery.toLowerCase();
 
-        return user.fullname.toLowerCase().indexOf(query) !== -1
-               ||
-               (user.lastname + ", " + user.firstname).toLowerCase().indexOf(query) !== -1
+		var full = user.fullname || '';
+		var first = user.firstname || '';
+		var last = user.lastname || '';
```

Comment:
Can you swap the tabs out for spaces here? After that it looks good—I'd say merge it.